doc_source/application-revisions-appspec-file.md

```diff
@@ -86,12 +86,10 @@ The following is a JSON template for an AppSpec file for an Amazon ECS deploymen
 				"Type": "AWS::ECS::Service",
 				"Properties": {
     			"TaskDefinition": "",
-    			"LoadBalancerInfo": [
```

Comment:
Load balancer info structure has to be a list. Today we support exactly 1 configuration only but it expects a list for future extendability just like load balancer configuration in ECS service configuration.

Reply:
Supplying a list results in the following error from CodeDeploy:
The AppSpec file must be valid JSON or YAML The input AppSpec file is a not well-formed yaml. The template cannot be parsed.